.github/workflows/nightly-link-checker.yml

```diff
@@ -0,0 +1,48 @@
+name: Nightly Job
+
+on:
+  schedule:
+    - cron: "1 5 * * *" # This schedule runs every night at 12:01 AM Eastern time (adjust according to your timezone)
```

Comment:
For testing you can run action on pull request, the same as run builds
```suggestion
# on:
#  schedule:
#    - cron: "1 5 * * *" # This schedule runs every night at 12:01 AM Eastern time (adjust according to your timezone)

on: pull_request
```

Reply:
Great. Will commit that once I finish local testing.